posthog/__init__.py

```diff
@@ -423,3 +423,6 @@ def _proxy(method, *args, **kwargs):
 
     fn = getattr(default_client, method)
     return fn(*args, **kwargs)
+
+class Posthog(Client):
```

Comment:
@neilkakkar Or maybe something like this where the global client situation still exists but we'll update docs to use this new pattern so new usage won't cause collisions

Reply:
yep exactly what I meant!